renderdoc/driver/gl/gl_common.cpp

```diff
@@ -415,6 +415,8 @@ void CheckExtensions(const GLHookSet &gl)
       if(ext == NULL || !ext[0] || !ext[1] || !ext[2] || !ext[3])
         continue;
 
+      RDCLOG("glGetStringi(eGL_EXTENSIONS, %i) = %s", i, ext);
```

Comment:
What's the reason for printing this information here? This will be called every time a new context is created I think, and since there are many hundreds of extensions available this will get very spammy in the logfile, which is already quite spammy for OpenGL.

There's a list of extensions printed once on replay that gets compacted on a couple of lines. Is there a need for that as well on capture?